recipe/meta.yaml

```diff
@@ -188,13 +187,13 @@ outputs:
         - openjpeg    # [build_platform != target_platform]
         - zlib        # [build_platform != target_platform]
       host:
-        - boost-cpp {{ boost_cpp }}
         - cairo
         - curl
         - fontconfig
         - freetype
         - gettext
         - glib
+        - libboost-headers
```

Comment:
And `libboost-headers` where there's not run-dep (and hence no run-requirement). Due to the layered build here, we should still verify that the link-check agrees.

Reply:
All builds have:
```
WARNING (poppler): dso library package conda-forge::libboost-1.82.0-hf0c313a_3 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
```
so indeed, we don't need `-devel` here.